br/pitr-known-issues.md

```diff
@@ -68,3 +68,13 @@ For the current version, it is recommended that you perform a snapshot backup af
 Issue: [#13304](https://github.com/tikv/tikv/issues/13304)
 
 When there is a large transaction, the log checkpoint lag is not updated before the transaction is committed. Therefore, the checkpoint lag is increased by a period of time close to the commit time of the transaction.
+
+## PITR is not compatible with the acceleration of adding indexes
+
+Issue: [#38045](https://github.com/pingcap/tidb/issues/38045)
+
+Currently, the [acceleration of adding indexes](#tidb_ddl_enable_fast_reorg-new-in-v630) is not compatible with PITR. When using index acceleration, you need to ensure that there are no PITR log backup tasks running in the background. Otherwise, unexpected behaviors might occur, including:
```

Comment:
```suggestion
Currently, the [acceleration of adding indexes](#tidb_ddl_enable_fast_reorg-new-in-v630) feature is not compatible with PITR. When using index acceleration, you need to ensure that there are no PITR log backup tasks running in the background. Otherwise, unexpected behaviors might occur, including:
```